pkg/armhelpers/azureclient.go

```diff
@@ -138,8 +138,8 @@ func NewAzureClientWithClientSecret(env azure.Environment, subscriptionID, clien
 	return getClient(env, subscriptionID, armSpt, adSpt)
 }
 
-// NewAzureClientWithClientCertificate returns an AzureClient via client_id and jwt certificate assertion
-func NewAzureClientWithClientCertificate(env azure.Environment, subscriptionID, clientID, certificatePath, privateKeyPath string) (*AzureClient, error) {
+// NewAzureClientWithClientCertificateFile returns an AzureClient via client_id and jwt certificate assertion
+func NewAzureClientWithClientCertificateFile(env azure.Environment, subscriptionID, clientID, certificatePath, privateKeyPath string) (*AzureClient, error) {
```

Comment:
NewAzureClientWithClientCertificateFile can call NewAzureClientWithClientCertificate and remove lines 168-177. This prevent code from diverging.